tailwind.themes.config.js

```diff
@@ -0,0 +1,42 @@
+export default [
+  {
+    light: {
+      primary: '#bb2c75',
+      'primary-content': '#fff',
+      secondary: '#818aa3',
+      'secondary-content': '#fff',
+      accent: '#e6a33a',
+      'accent-content': '#fff',
+      neutral: '#3d4451',
+      'neutral-content': '#fff',
+      'base-100': '#fff',
+      'base-200': '#f9fafb',
+      'base-300': '#d1d5db',
+      'base-content': '#292929',
+      info: '#2094f3',
+      success: '#009485',
+      warning: '#ff9900',
+      error: '#ff5724',
+    },
+  },
+  {
+    dark: {
+      primary: '#bb2c75',
+      'primary-content': '#fff',
+      secondary: '#818aa3',
+      'secondary-content': '#fff',
+      accent: '#e6a33a',
+      'accent-content': '#fff',
+      neutral: '#3d4451',
+      'neutral-content': '#fff',
+      'base-100': '#3b424e',
+      'base-200': '#2a2e37',
+      'base-300': '#16181d',
+      'base-content': '#ebecf0',
+      info: '#2094f3',
```

Comment:
issue (blocking): Currently the combination of our primary color with base-200 is resulting in a poor contrast. We should therefore use darker background accents (current base-300 and below would be good) or a lighter primary color.

Reply:
Adjusted primary color as @moritzmurauer suggested